drivers/src/hand_off.rs

```diff
@@ -280,8 +280,11 @@ pub struct FirmwareHandoffTable {
     /// Index of LdevId Certificate Signature S Component in the Data Vault.
     pub ldevid_cert_sig_s_dv_hdl: HandOffDataHandle,
 
-    /// IDevID public key
-    pub idev_dice_pub_key: Ecc384PubKey,
+    /// IDevID ECDSA public key
+    pub idev_dice_ecdsa_pub_key: Ecc384PubKey,
+
+    /// IDevID MLDSA public key address in DCCM
+    pub idev_dice_mldsa_pub_key_load_addr: u32,
```

Comment:
Since the FHT is a fixed structure that's shared between fw layers, I think we should stick with fixed-size types. If we need to change to u64 in the future we'll have to make that an explicit decision and make sure all fw layers handle it properly.